eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -172,6 +172,7 @@ jobs:
         displayName: 'Install dependencies'
 
       - script: |
+          xcrun simctl delete unavailable
```

Comment:
what is the step for?

Reply:
This is used to close any simulators currently open on the CI virtual machine.